frontend/src/components/CommentList/CommentList.test.jsx

```diff
@@ -1,7 +1,7 @@
 import { render, screen } from '@testing-library/react';
 import { CommentList } from './CommentList';
-import { avatar1 } from '../../../mocks/images/avatars/sample-avatar1.jpg';
-import '../../../setupTests';
+import { avatar1 } from '../../mocks/images/avatars/sample-avatar1.jpg';
```

Comment:
Nie musisz poprawiać, ale pamiętaj, że mamy też ścieżki bazwzględne z "src", więc tutaj będzie dobrze:
```suggestion
import { avatar1 } from 'mocks/images/avatars/sample-avatar1.jpg';
```